### ISSUE TITLE
\MakeSentenceCase and accented characters

### ISSUE THREAD

**Issue by moewew** [plk/biblatex#408]
Consider the following MWE

```
\documentclass{article}
\usepackage[T1]{fontenc}
\usepackage[style=numeric,backend=biber]{biblatex}

\DeclareFieldFormat{titlecase}{%
  \MakeSentenceCase{#1}%
}

\usepackage{filecontents}
\begin{filecontents}{\jobname.bib}
@book{test,
    title={{\'a}{\'i}}
}
\end{filecontents}

\addbibresource{\jobname.bib}
\begin{document}
\MakeSentenceCase{{\'a}{\'i}}
\nocite{test}
\printbibliography
\end{document}
```

which fails with an error about `Argument of \T1\' has an extra }`.

It seems as though this has to do with how the character "í" is saved in the `.bbl` if the target document can't deal with UTF-8 (with the MWE I get `\'{a}\'\i{}` and indeed `\MakeSentenceCase{\'{a}\'\i{}}` doesn't work, but both `\'{a}\'{\i}` and `\'{a}\'{i}` would be fine, I guess).

(See also http://tex.stackexchange.com/q/304565/)

**Comment by plk** [comment]
Ok, I'll put a fix in for this in 2.5 since it seems to really only apply in edge cases.

**Comment by moewew** [comment]
Thank you. 

Sorry for abusing this issue here, but do you have any thoughts about this problem with Bob and the name counters (http://tex.stackexchange.com/q/303296/)?
